src/ducks/groups/helpers.spec.js

```diff
@@ -0,0 +1,36 @@
+import { buildVirtualGroups } from './helpers'
+
+describe('buildVirtualGroups', () => {
+  it('should generate a virtual group if there are two accounts with the same type', () => {
+    const accounts = [
+      { _id: '1', type: 'checkings' },
+      { _id: '2', type: 'checkings' }
+    ]
+
+    const virtualGroups = buildVirtualGroups(accounts)
+    const expected = [
+      {
+        _id: 'checkings',
+        _type: 'io.cozy.bank.groups',
+        label: 'checkings',
+        accounts: {
+          data: accounts
+        },
```

Comment:
Since this is an internal structure of `cozy-client`, I think it would be better if it was hidden in a function, maybe as a helper in utils/cozy-client that could later be exported in `cozy-client`. The goal being that at some point if cozy-client decides to change the structure of its relationships, we should have our test that fail. WDYT ?

Reply:
This is related to https://github.com/cozy/cozy-banks/blob/master/src/utils/client.js#L62

Reply:
You think about a function like `associateDocuments(originalDocument, associationName, associatedDocuments)` ?